lib/components/viewers/stop-schedule-table.tsx

```diff
@@ -53,7 +54,14 @@ const TimeCell = styled.td`
  * Table showing scheduled departure times for the specified stop organized
  * chronologically.
  */
-class StopScheduleTable extends Component {
+class StopScheduleTable extends Component<{
+  date: any
+  // TODO TYPESCRIPT shared types
+  findStopTimesForStop: (options: any) => void
+  showBlockIds: boolean
+  // TODO TYPESCRIPT shared types
+  stopData: any
```

Comment:
I've done this in 1da36d19fbd6109249facc2d9a6c42fb3b4dc9e6, but it doesn't feel great... and I'm not quite comfortable using the new type all over the place because I think it should live in the new types package

Reply:
Oh... I meant to place the type for the props of the `StopScheduleTable` component to a local type variable, but since you did the type for `stopData`, keep that one where it is.